web.config

```diff
@@ -5,7 +5,9 @@
       <add name="httpPlatformHandler" path="*" verb="*" modules="httpPlatformHandler" resourceType="Unspecified" />
     </handlers>
     <httpPlatform processPath="%JAVA_HOME%\bin\java.exe"
-        arguments="-Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\private-beta-invitation-service-0.0.1.jar&quot;">
+                  stdoutLogEnabled="true"
+                  stdoutLogFile="%HOME%\LogFiles\stdout"
+                  arguments="-javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.0.2.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\private-beta-invitation-service.jar&quot;">
```

Comment:
Are you not missing out the `/lib/applicationinsights-agent-2.0.2.jar` in source code? And possibly `AI-Agent.xml` config file too

Reply:
ohh I missed app insights logging part thought we are making this change only for making the pipeline green.App insights can be integrated as a separate PR but up to you guys.

Reply:
PR is super tiny. including tiny xml with HTTP enabled will still make a wee pr i'm happy about. and non-breaking change too.

Reply:
Removed appinsights